layouts/_default/events.yaml

```diff
@@ -0,0 +1,16 @@
+{{- $events := (.Site.GetPage "events").Pages -}}
+{{- range sort $events ".Date" "asc" -}}
+{{- $images := .Resources.ByType "image" -}}
+{{- $feature := $images.GetMatch .Params.feature }}
+- title: {{ .Params.title }}
```

Comment:
can you sanitize the string input? When the title has a colon, it breaks YAML convention. e.g `"Example: Title is now stringified"` vs `Example: Title is not stringified`

Reply:
Fixed! Also fixed tags and location tags since they are somewhat free text and could lead to similar issues.